### ISSUE TITLE
Filter update not working

### ISSUE THREAD

**Issue by l3enjay** [notablackbear/poe_qolV2#33]
I'm having an issue with the filter update not working correctly - 
![image](https://user-images.githubusercontent.com/44820756/93666050-e7c6c280-fa72-11ea-927d-0404e7e6ac72.png)
I believe this is due to a typo in 'Pernach', assuming some mapping is being done somewhere, as I have the POEQOL_Base.filter file to try and remedy this error and it hasn't fixed it

**Comment by jinkai** [comment]
This is the same issue I'm running into - everything else seems to be working fine, but after refreshing via the overlay and then reloading the filter, then I see this.

**Comment by mgerni** [comment]
Also having this issue.

**Comment by EgrenB3S23** [comment]
This was an issue with a version of Neversink's filter right at league launch, unrelated to PoEQoL. We were subscribed to Neversink's filter (=no edits after it left Neversink's hands) when the league started, and it wouldn't load, giving the same error message. A few hours later the filter had been fixed.

I suspect the filter you're trying to use was based on that faulty Neversink filter version.

**Comment by jinkai** [comment]
So I deleted the filters I had, and let PoEQoL use the one it came with, and I still get this issue. Is there somewhere to get a fixed filter?

**Comment by RuneBarrett** [comment]
It works if you remove the two instances of "Pernarch" from the filter

**Comment by Memeraldick** [comment]
> It works if you remove the two instances of "Pernarch" from the filter

You need to remove them everytime filter automaticaly updates so thats not much of a fix

**Comment by jinkai** [comment]
Yeah, I just tried removing it from both my filter and the one bundled with the app, and I still get it. I guess i can just have my notepad++ script replace "pernarch"  (1 space at the end) with nothing.

**Comment by sorlix92** [comment]
Its the software not the filter that makes this issue. Just replace the "Pernarch" with "Pernach" It would be a 1 min fix but i guess the programmer is too busy.

**Comment by sclient** [comment]
> 
> 
> > It works if you remove the two instances of "Pernarch" from the filter
> 
> You need to remove them everytime filter automaticaly updates so thats not much of a fix

Hmm, Ive managed to manually edit out the entries in the one bundled with the filter, and when you set the filter= to blank (on the new version, not the release one), it will copy that filter into the poe folder.
